packages/hdwallet-provider/test/provider.test.ts

```diff
@@ -304,5 +290,50 @@ describe("HD Wallet Provider", function () {
       const number = await web3.eth.getBlockNumber();
       assert(number === 0);
     });
+
```

Comment:
I think the [provider tests setups](https://github.com/trufflesuite/truffle/blob/45e3dbe3738699997d6f0985e352ccdec6477560/packages/hdwallet-provider/test/provider.test.ts#L143) should use `url` instead of `providerOrUrl` to reflect the newer interface as tests are used to document usage.

Reply:
There are a few other spots where `providerOrUrl` is used when `url` would suffice.

Reply:
good idea, changed them in the test!